tests/test_makeSurveyPropertyMaps.py

```diff
@@ -220,9 +220,9 @@ def testPropertyMapCreationEmptyInputMap(self):
         # Replace the input map with an empty one.
         with warnings.catch_warnings():
             warnings.simplefilter("ignore")
-            self.input_map_dict[0].input_map = hsp.HealSparseMap.make_empty_like(
-                self.input_map_dict[0].input_map
-            )
+            self.input_map_dict[0] = MockInputMapReference(hsp.HealSparseMap.make_empty_like(
+                self.input_map_dict[0].inMemoryDataset
+            ), **self.input_map_dict[0].dataId)
```

Comment:
So you didn't like the way I was just hacking in the value? 😄

Reply:
It's not a question of "like" it's a problem with InMemoryDatasetHandle being read-only.